### PR TITLE
cli: fix unbound variables, formatting

### PR DESCRIPTION
**Pull request by joukost**
Subcommands to update integration or integration endpoint fail with an `UnboundLocalError` unless config variables are also supplied.

Also removed redundant parentheses.

### CODE REVIEW

File: aiven/client/cli.py [hunk 1]
```diff
@@ -1317,9 +1317,9 @@ def service__integration_endpoint_create(self):
     @arg.json
     def service__integration_endpoint_update(self):
         """Update a service integration endpoint"""
+        endpoint_id = getattr(self.args, "endpoint-id")
```
Comment (rikonen) [inline aiven/client/cli.py:1320]: Fwiw I think it would be better to name the argument `endpoint_id` to avoid the use of `getattr()`.

**Review by rikonen — APPROVED** [review]

**Review by rikonen — CHANGES_REQUESTED** [review]
Looks like pylint version has changed at this is complaining about some existing issue

File: aiven/client/cli.py (hunk 1 above)
Comment (joukost) [reply aiven/client/cli.py:1322]: Makes sense, it's positional anyway. I'll change it...

**Comment by joukost** [comment]
Apparently my local `yapf` didn't agree too well either.

**Review by rikonen — APPROVED** [review]

**Pull request merged**